### PR TITLE
feature: added toast

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added toast
- success toast
- warning toast
- error toast

### CODE REVIEW

File: components/toast/toast.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+      href="../../assets/favicon/favicon-red-black.ico"
+      type="image/x-icon"
+    />
+    <link rel="stylesheet" href="../../css/main.css" />
+    <link rel="stylesheet" href="../../css/utilities.css" />
+    <link rel="stylesheet" href="../../css/introduction.css" />
+    <link rel="stylesheet" href="../avatar/avatar.css" />
+    <link rel="stylesheet" href="../button/button.css" />
+    <link rel="stylesheet" href="../typography/typography.css" />
+    <link rel="stylesheet" href="../toast/toast.css" />
```
Comment (junaidshaikh-js) [inline components/toast/toast.html:28]: Instead of linking multiple CSS files, import all files in one file and link that to the page.

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

File: components/toast/toast.html (hunk 1 above)
Comment (omgaikwad) [reply components/toast/toast.html:28]: Yes I am going to do that at the end, after completing all the components, thanks for suggestion.

**Review by ZuberDunge — COMMENTED** [review]
looks good to me.

**Pull request merged**